Content.Server/StationEvents/Components/SentienceTargetComponent.cs

```diff
@@ -7,4 +7,7 @@ public sealed partial class SentienceTargetComponent : Component
 {
     [DataField("flavorKind", required: true)]
     public string FlavorKind = default!;
+
+    [DataField("weight")]
```

Comment:
```suggestion
    [DataField]
```
DataField will automatically use the name of the field with the first letter lowercased. You only need to specify the tag if for some weird reason you need to call the field something else in YAML prototypes.

There are still lots of examples in the code of this not being done, but that's because it's a semi-recent feature and old code hasn't been updated.